data/spec_adopters.yml

```diff
@@ -13,6 +13,10 @@ adopters:
       homepage_url: https://www.meetup.com/MadrasJUG/
       logo: mjug.svg
       logo_white: mjug.svg
+    - name: JUG-PE
+      homepage_url: https://jugpe.com.br/
+      logo: jugpe.svg
```

Comment:
This looks like it should be jugpe.png, not svg

Reply:
Sorry about that, fixed